vaadin-testbench-shared/src/main/java/com/vaadin/testbench/TestBenchElement.java

```diff
@@ -177,6 +177,12 @@ public void click() {
             // Avoid strange "element not clickable at point" problems
             callFunction("click");
         } catch (Exception e) {
+            if (e.getMessage()
+                    .contains("Inspected target navigated or closed")) {
+                // This happens with chromedriver although e.g. navigation
+                // succeeds
+                return;
```

Comment:
Did I understand correctly that if this error occurs, the action is already completed correctly, and it is safe to ignore the exception?

Reply:
That is what happens in the flow tests. The navigation completes, then an exception is thrown and because of the logic in TestBenchElement.click, the action is retried and fails as navigation did happen